demos/dev-demo/src/XDEFIWalletAdapterDemo.tsx

```diff
@@ -0,0 +1,257 @@
+import type { ReactNode } from 'react';
+import { useEffect, useMemo, useState, useCallback } from 'react';
+import { TronLinkAdapter } from '@tronweb3/tronwallet-adapters';
```

Comment:
@HoangVD2 you need to create a XDEFIAdapter  (which will be similar to TronLinkAdapter) instead of using TronLinkAdapter here;

Reply:
just updated @davidp94